main.go

```diff
@@ -610,6 +632,28 @@ func main() {
 	ctx = logger.Set(ctx, "main")
 	defer l.Info("Done.")
 
+	cfg := config.NewService()
+	if err := cfg.Load(); err != nil {
+		l.Panicf("Failed to load config: %+v", err)
+	}
+	if err := cfg.Update(func(s *config.Service) error {
+		ds := s.Config.Services.Telemetry.DataSources
+		pmmdb := ds.PmmDBSelect
+
+		pmmdb.Credentials.Username = *postgresDBUsernameF
+		pmmdb.Credentials.Password = *postgresDBPasswordF
+		pmmdb.DSN.Scheme = "postgres" //TODO: should be configurable
+		pmmdb.DSN.Host = *postgresAddrF
+		pmmdb.DSN.DB = *postgresDBNameF
+		q := make(url.Values)
+		q.Set("sslmode", "disable")
+		pmmdb.DSN.Params = q.Encode()
+
+		return nil
+	}); err != nil {
+		l.Panicf("Failed to update config: %+v", err)
+	}
```

Comment:
it just complicates logic
```suggestion
		ds := cfg.Config.Services.Telemetry.DataSources
		pmmdb := ds.PmmDBSelect

		pmmdb.Credentials.Username = *postgresDBUsernameF
		pmmdb.Credentials.Password = *postgresDBPasswordF
		pmmdb.DSN.Scheme = "postgres" //TODO: should be configurable
		pmmdb.DSN.Host = *postgresAddrF
		pmmdb.DSN.DB = *postgresDBNameF
		q := make(url.Values)
		q.Set("sslmode", "disable")
		pmmdb.DSN.Params = q.Encode()
```

Reply:
it had a purpose, it needed to be executed at a specific time. as of now, it is not needed. will simplify it